target/linux/mediatek/patches-5.15/102-mt7629-enable-arch-timer.patch

```diff
@@ -1,10 +1,10 @@
 --- a/arch/arm/mach-mediatek/Kconfig
 +++ b/arch/arm/mach-mediatek/Kconfig
```

Comment:
This patch is upstreamed:

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.15.46&id=ffea838686b82fbb2801cdfad6ba5309d15c032d

The auto-refresh applies this to mt7623 and...

Reply:
Ugh… quilt trying to be smarter than it should, I guess. Will fix…